ansible_collections/arista/avd/roles/eos_designs/python_modules/inband_management/avdstructuredconfig.py

```diff
@@ -162,7 +211,8 @@ def _inband_management_parent_vlans(self) -> dict:
             if (vlan := peer_facts.get("inband_mgmt_vlan")) is None:
                 continue
 
-            if (subnet := peer_facts.get("inband_mgmt_subnet")) in subnets:
+            subnet = {"ipv4": peer_facts.get("inband_mgmt_subnet"), "ipv6": peer_facts.get("inband_mgmt_ipv6_subnet")}
+            if subnet in subnets:
```

Comment:
I think you need to maintain two lists so you can check each AF. Otherwise you will get some weird results if some devices have one, some the other and some both.

Reply:
Fixed now... i hope.